frontend/public/components/service-instance.tsx

```diff
@@ -73,7 +104,20 @@ const ServiceInstanceDetails: React.SFC<ServiceInstanceDetailsProps> = ({obj: si
   </React.Fragment>;
 };
 
-const pages = [navFactory.details(ServiceInstanceDetails), navFactory.editYaml(), navFactory.events(ResourceEventStream)];
+const ServiceBindingsDetails: React.SFC<ServiceBindingsDetailsProps> = ({obj: si}) => {
+  const bindingCreateHandler = createBinding(null, si).callback;
+  const bindingFilters = {selector: {field: 'spec.instanceRef.name', values: new Set(_.map(si, 'name'))}};
+
+  return <ServiceBindingsPage canCreate={true} createHandler={bindingCreateHandler} namespace={si.metadata.namespace} filters={bindingFilters} autoFocus={false} showTitle={false} />;
```

Comment:
I'd use the helper here, too.

```suggestion
  return <ServiceBindingsPage canCreate={true} createHandler={() => goToCreateBindingPage(si)} namespace={si.metadata.namespace} filters={bindingFilters} autoFocus={false} showTitle={false} />;
```